utils.py

```diff
@@ -16,6 +16,9 @@
 import shutil
 import yaml
 
+import numpy as np
```

Comment:
It is not included in `requirements.txt`

Reply:
Good catch! I forgot to push this change. Fixed in https://github.com/spinalcordtoolbox/manual-correction/pull/15/commits/06dba920ec84786573596de77cd4da943b1c8e8f